mysql-test/suite/galera/t/galera_sst_xtrabackup-v2_pxc_encrypt.test

```diff
@@ -2,12 +2,64 @@
 # This test checks the pxc-encrypt-cluster-traffic option (auto SSL config).
 # Initial SST happens via xtrabackup, so there is not much to do in the body of the test
 #
+# Also tests certs that are specified using relative paths (certs are in the datadir)
+#
 
 --source include/big_test.inc
---source include/galera_cluster.inc
 --source include/have_innodb.inc
+--source include/galera_cluster.inc
+--source include/force_restart.inc
+
+# --------------------------------
+# Test 1: Just check that the cluster started correctly.
+# --------------------------------
+--let $wait_condition = SELECT VARIABLE_VALUE = 2 FROM performance_schema.global_status WHERE VARIABLE_NAME = 'wsrep_cluster_size';
+--source include/wait_condition.inc
 
 SELECT 1;
 
---let $wait_condition = SELECT VARIABLE_VALUE = 2 FROM INFORMATION_SCHEMA.GLOBAL_STATUS WHERE VARIABLE_NAME = 'wsrep_cluster_size';
+# --------------------------------
+# Test 2: Test that the server starts up with SSL certs specified
+# using relative paths (place the certs in the datadir)
+# --------------------------------
+
+# Create cnf file with relative paths
+--let $RELATIVE_SSL_CERTS_FILE = $MYSQLTEST_VARDIR/tmp/galera_sst_xtrabackup-v2_pxc_encrypt.test.cnf
+--copy_file $MYSQLTEST_VARDIR/my.cnf $RELATIVE_SSL_CERTS_FILE
+--append_file $RELATIVE_SSL_CERTS_FILE
+	[mysqld]
+	ssl-ca=cacert.pem
+	ssl-key=server-key.pem
+	ssl-cert=server-cert.pem
+EOF
```

Comment:
The original test code didn't run the SST code which is where it fails.  Although this will test that galera picks up the certs using a relative (to the datadir) path.
I prefer leaving the cnf file as [mysqld] so any server using this cnf will be using the relative cert paths.

Reply:
The code to remove the grastate.dat was added to force the SST.